### PR TITLE
secp256k1 ecdsa

### PR DESCRIPTION
**Pull request by davidiw**

### CODE REVIEW

File: aips/aip-49.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+updated (*optional): <mm/dd/yyyy>
+---
+
+# AIP-49 - Secp256k1 Ecdsa for Transaction Authentication
+  
+Despite our desire to see more diverse key algorithms supported in hardware cryptographic platforms, the primary Aptos key algorithm, Ed25519, has yet to be adopted broadly across the ecosystem. Secp256k1 Ecdsa remains the incument and is broadly supported. This AIP introduces support Secp256k1 Ecdsa as a transaction authenticator for Aptos.
+
+## Summary
+
+In Aptos, each transacation contains a transaction authenticator that includes a signature and a public key, while the transaction itself contains the sender of the transaction. To verify that a transaction is properly signed, the verifier validates that the public key verifies the signature across the transaction and that the hash of the public key is stored on-chain in a hashed form under the account. By completing this verification, the verifier can be certain that the owner of the account indeed authorizes this transaction. This AIP adds support for Secp256k1 Ecdsa for transaction authentication.
```
Comment (mstraka100) [inline aips/aip-49.md:19]: typo: "transacation"

**Comment by alinush** [comment]
Can we include @mstraka100's `secp256r1` into this?

File: aips/aip-49.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Interface
+created: 09/28/2023
+updated (*optional): <mm/dd/yyyy>
+---
+
+# AIP-49 - Secp256k1 Ecdsa for Transaction Authentication
+  
+Despite our desire to see more diverse key algorithms supported in hardware cryptographic platforms, the primary Aptos key algorithm, Ed25519, has yet to be adopted broadly across the ecosystem. Secp256k1 Ecdsa remains the incument and is broadly supported. This AIP introduces support Secp256k1 Ecdsa as a transaction authenticator for Aptos.
```
[the remark this answers is not in the document]
Comment (alinush) [reply aips/aip-49.md:15]: ECDSA should be capitalized everywhere.

[the remark this answers is not in the document]
Comment (alinush) [reply aips/aip-49.md:15]: `secp256k1` should NOT be capitalized and probably highlighted with `code syntax`.

**Review by movekevin — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

**Comment by davidiw** [comment]
Wow @thepomeranian, you're incredible!!! Thank you!

**Pull request merged**